make.js

```diff
@@ -949,14 +956,70 @@ function getFormattedDatatypeDescription(prefix, datatype) {
     return output;
 }
 
-function getFormattedCallDescription(apiName, call) {
-    if (apiName in xmlRefDocs) {
-        var apiRefDocs = xmlRefDocs[apiName];
-        if (call.name in apiRefDocs.calls) {
-            return appendToXmlDocComment("///", apiRefDocs.calls[call.name].summary);
+function getSeeAlso(call) {
+    if (call.seeAlso == undefined || call.seeAlso.length == 0) {
+        return "";
+    }
+
+    var result = "See also ";
+    for (var i in call.seeAlso) {
+        result += "PlayFab" + call.seeAlso[i].replace(/"/g, "'").replace("/", "") + "Async";
+        if (i != call.seeAlso.length - 1) {
+            result += ", ";
         }
     }
-    return "/// " + call.name + " documentation not found in XmlRefDocs."
+
+    return result;
+}
+
+function getCallDoc(apiName, call) {
+    var asyncName = "PlayFab" + apiName + call.name + "Async";
+    if (asyncName in xmlRefDocs) {
+        var docForCall = xmlRefDocs[asyncName];
+        return docForCall;
+    }
+
+    if (asyncName == "PlayFabAdminGetCloudScriptTaskInstanceAsync") {
+        asyncName = "PlayFabAdminGetCloudScriptTaskInstanceAsync";
+    }

```

Comment:
why is a special case needed for this API?  Also, prefer '===' when comparing in js

Reply:
oh dead code.  i'll remove